compiler/orchestrator_runtime/pash_init_setup.sh

```diff
@@ -23,10 +23,10 @@ export pash_daemon_communicates_through_unix_pipes_flag=0
 export pash_speculative_flag=0
 export show_version=0
 export distributed_exec=0
+export local_annotations=0
```

Comment:
Could you use the naming convention from above and move the flag close to the other ones? `pash_checking_local_annotations`